www/views/page/content/list.php

```diff
@@ -29,11 +31,34 @@ function cap($text, $len)
     echo "<div class='w-100 text-right'>Filter: ";
     echo "<a class='" . ($filter == 0 ? 'text-success' : '') . "' href='?action=list&filter=0'>alles</a> | ";
     echo "<a class='" . ($filter == 1 ? 'text-success' : '') . "' href='?action=list&filter=1'>klaar</a> | ";
-    echo "<a class='" . ($filter == 2 ? 'text-success' : '') . "' href='?action=list&filter=2'>klaar & nog niet nagekeken</a>";
+    if ($role == 2) {
+    	echo "<a class='" . ($filter == 2 ? 'text-success' : '') . "' href='?action=list&filter=2'>klaar & nog niet nagekeken</a> ";
```

Comment:
Ik hoor ook vaak verzoeken voor "klaar en nog niet naargekeken _door mij_"

Reply:
Ik heb dit al in klaar en nagekeken gestopt. Als je klaar en nagekeken aan hebt, krijg je de stukjes die jij nog niet hebt nagekeken en die nog geen drie keer zijn nagekeken.

Reply:
Misschien moeten we die 3 nog eens een instelbaar ding maken voor beheerders.